### DISCUSSION TITLE
Github Student Developer Pack renewal application got rejected twice!

### DISCUSSION THREAD

**Discussion by abzaidi** [community/community#139965] [GitHub Education]
### Hi everyone,

Hello everyone, 
I have been trying to renew my github developer pack application but for some reason it keeps getting rejected and the response I get is something that does not explain what's the reason behind my rejection. I am still an active student with the expiry date clearly mentioned on my university card which I have taken picture of both times while applying. If there is anyone from github support staff reading this, kindly let me know the reason behind my rejection.

This is the response I got both times while applying for renewal:
![Capture](https://github.com/user-attachments/assets/d05cbff6-3698-408f-ad8b-34da817ad43f)

**Comment by AidenJamesYt** [comment]
Hi!

In the response it mentions that you don't appear to be near any school affiliated with your student ID. If you are a remote student, make sure that its clearly stated!

If you have a VPN on, turn it off, if possible. If you use a school-issued device, it may have a VPN that cannot be turned off. If this is the case, try applying from a personal device, not owned by the school/university.


If you continue to have issues, I recommend reaching out to GitHub's Support Team.

Here is a link to [Github's Support Website](https://support.github.com/).

Here is a link to open a [Support Ticket](https://support.github.com/request).

If you need further assistance, let me know! :D

**Reply by abzaidi** [reply]
Thanks for the reply, does this mean I have to apply while being on campus?

**Reply by AidenJamesYt** [reply]
If possible, yes
